docs/terms/action.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Action 
+id: action 
+hoverText: A Python class with a run method that takes the result of validating a Batch against an Expectation Suite and does something with it
+---
+
+A Validation Action is a Python class with a run method that takes the result of validating a Batch
```

Comment:
not sure if you wanted this in this branch.

Reply:
These tweaks were only made in this branch because the Checkpoint doc has a link to the technical term "Action" -- the PR that updates "Validation Action" reduces that technical term to "Action", so these tweaks were made so that the Checkpoint page wouldn't have to be adjusted when the PR for the "Action" technical term is merged.